### PR TITLE
remove unnecessarily validation

### PR DESCRIPTION
**Pull request by deads2k**
The old configuration is only used and accepted for integration tests.  Separate remaining validation to its point of use (CLI and networking), the remove the remaining conflicting validation.

/assign @mfojtik

### CODE REVIEW

File: pkg/cmd/openshift-controller-manager/cmd.go [hunk 1]
```diff
@@ -146,17 +145,6 @@ func (o *OpenShiftControllerManager) RunControllerManager() error {
 	if err != nil {
 		return err
 	}
-
-	validationResults := validation.ValidateMasterConfig(masterConfig, nil)
```
Comment (mfojtik) [inline pkg/cmd/openshift-controller-manager/cmd.go]: why is this not needed?

File: pkg/cmd/openshift-sdn/cmd.go [hunk 2]
```diff
@@ -9,6 +9,8 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/openshift/origin/pkg/network/common/networkvalidation"
```
Comment (mfojtik) [inline pkg/cmd/openshift-sdn/cmd.go]: nit: move down, sort imports

File: pkg/network/common/networkvalidation/network.go [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/openshift/library-go/pkg/config/validation"
+	"github.com/openshift/library-go/pkg/crypto"
+	configapi "github.com/openshift/origin/pkg/cmd/server/apis/config"
+)
+
+func ValidateInClusterNetworkNodeConfig(config *configapi.NodeConfig, fldPath *field.Path) validation.ValidationResults {
```
Comment (mfojtik) [inline pkg/network/common/networkvalidation/network.go]: @deads2k this is new or where does this came from?

File: pkg/cmd/openshift-controller-manager/cmd.go (hunk 1 above)
Comment (deads2k) [reply pkg/cmd/openshift-controller-manager/cmd.go]: > why is this not needed?

never valid except from integration test.

File: pkg/network/common/networkvalidation/network.go (hunk 3 above)
Comment (deads2k) [reply pkg/network/common/networkvalidation/network.go]: it was originally a move that needed some package updates.

**Pull request merged**